src/main/java/org/wso2/securevault/secret/SecretManager.java

```diff
@@ -230,6 +277,31 @@ public String getSecret(String alias) {
         return parentRepository.getSecret(alias);
     }
 
+    /**
+     * Returns the encrypted value corresponding to the given secretAnnotation name where the
+     * secretAnnotation consists of provider type, repository type and the alias.
+     *
+     * @param provider   Provider type.
+     * @param repository Repository type.
+     * @param alias      Alias to be resolved.
+     * @return If there is a secret , otherwise , alias itself.
+     */
+    private String getSecret(String provider, String repository, String alias) {
+
+        if (!providers.containsKey(provider)) {
+            log.error("Provider type in the annotation does not match with the configured providers. " +
+                    "Returning the alias itself.");
+            return alias;
+        }
+        if (!secretRepositories.containsKey(repository)) {
+            log.error(
```

Comment:
Can we reformat so that this log looks nice ?

Reply:
Reformatted